### DISCUSSION TITLE
New files not showing in Zettlr

### DISCUSSION THREAD

**Discussion by jneug** [Zettlr/Zettlr#3307] [Q&A] [answered]
Somehow new files are not shown in Zettlr, after I save them. Only after restarting the app, the files are visible. This happens both for files created and saved in Zettlr and in other applications.

Is there a setting I'm missing?

**Comment by nathanlesage** [comment, marked as answer]
There was a bug causing this. It should've been fixed in the most recent version 2.2.4. Could you please check that?

**Reply by jneug** [reply]
Thanks, seems to work now.
